task/configure-build/0.1/configure-build.yaml

```diff
@@ -40,35 +36,12 @@ spec:
         echo "App Studio Configure Build"
 
         DEST=/workspace/source/.dockerconfigjson
-        DEF=/secret/default-push-secret/.dockerconfigjson
-        AUTH=/workspace/registry-auth/.dockerconfigjson
-        TMP=$(mktemp)
-        echo '{}' > $DEST
-        # Set lowest priority on default shared secret
-        FILES="$DEF"
-        # Use secrets from serviceAccount
-        cd /tekton/creds-secrets
-        for file in $(ls); do
-          if [ -f "$file/.dockerconfigjson" ]; then
-            FILES="$FILES $file/.dockerconfigjson"
-          elif [ -f "$file/.dockercfg" ]; then
-            # convert format from .dockercfg to .dockerconfigjson
-            newformat=$(mktemp)
-            jq '{"auths": .}' $file/.dockercfg > $newformat
-            FILES="$FILES $newformat"
-          fi
-        done
```

Comment:
This seems to remove more than just the registry-auth secret. I'm probably missing some context, why are these being removed as well?

Reply:
ok, updating

Reply:
Makes sense, thanks